src/azure-cli-core/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+2.17.1
+++++++
+* Minor fixes
```

Comment:
no change

Reply:
2.17.1
++++++
* No change

like this?